box-share-sdk/src/main/java/com/box/androidsdk/share/sharerepo/ShareRepo.java

```diff
@@ -202,15 +198,15 @@ public void fetchCollaborations(BoxCollaborationItem boxCollaborationItem) {
      * Fetch supported features.
      */
     public void fetchSupportedFeatures() {
-        handleTaskAndPostValue(mController.getSupportedFeatures(), mSupportedFeature);
+        handleTaskAndPostValue(mController.getSupportedFeatures(), mSupportedFeatures); //this api might not be working as intended.
```

Comment:
Can we elaborate on the content. What is expected and what are you seeing the causes you to write this comment. It would also be nice to have this comment just above the method. Allows for more space and readability.